Default.sublime-keymap

```diff
@@ -1,28 +1,91 @@
-[
-    { "keys": ["f1"],               "command": "open_ceedling_file", "args": {"option": "config"} },
-
-    { "keys": ["f5"],               "command": "rake", "args": {"cmd": ["rake", "build:release"], "file_regex": "^(...*?):", "line_regex": "^...*?:([0-9]*)"} },
-
-    { "keys": ["f7"],               "command": "rake", "args": {"tasks": ["test:$file_name"] } },
-    { "keys": ["ctrl+f7"],          "command": "rake", "args": {"tasks": ["test:delta"] } },
-    { "keys": ["super+ctrl+b"],     "command": "rake", "args": {"tasks": ["test:delta"] } },
-    { "keys": ["ctrl+alt+f7"],      "command": "rake", "args": {"tasks": ["test:all"] } },
-    { "keys": ["ctrl+shift+b","o"], "command": "rake", "args": {"tasks": ["test:open"], "file_regex": "^(...*?):", "line_regex": "^...*?:([0-9]*)"} },
-    
-    { "keys": ["ctrl+alt+period"],  "command": "open_ceedling_file", "args": {"option": "test_and_source"} },
-    { "keys": ["ctrl+period"],      "command": "open_ceedling_file", "args": {"option": "next"} },
-    { "keys": ["ctrl+alt+h"],       "command": "open_ceedling_file", "args": {"option": "header"} },
-    { "keys": ["ctrl+alt+s"],       "command": "open_ceedling_file", "args": {"option": "source"} },
-    { "keys": ["ctrl+alt+t"],       "command": "open_ceedling_file", "args": {"option": "test"} },
-    
-    { "keys": ["ctrl+alt+m"],       "command": "rake", "args": {"tasks": ["module:create[$file]"], "file_regex": "^(...*?):", "line_regex": "^...*?:([0-9]*)"} },
-    
-    { "keys": ["ctrl+shift+"],      "command": "create_ceedling_module", "args": {"option": "work"} },
-
-    { "keys": ["ctrl+alt+i","o"],   "command": "rake", "args": {"tasks": ["ide:open"]} },
-    { "keys": ["f5"],               "command": "rake", "args": {"tasks": ["ide:make"], "file_regex": "(\\\\w+\\\\.[chp])", "line_regex": "^...*?:([0-9]*)"} },
-    { "keys": ["ctrl+f5"],          "command": "rake", "args": {"tasks": ["ide:build"], "file_regex": "(\\\\w+\\\\.[chp])", "line_regex": "^...*?:([0-9]*)"} },
-
-    { "keys": ["ctrl+alt+x"],       "command": "rake", "args": {"tasks": ["app_clean"], "file_regex": "^(...*?):", "line_regex": "^...*?:([0-9]*)"} },
-    { "keys": ["ctrl+alt+shift+x"], "command": "rake", "args": {"tasks": ["clobber"], "file_regex": "^(...*?):", "line_regex": "^...*?:([0-9]*)"} }
-]
+[
+
+    {
+        "keys": [
+            "super+ctrl+h"
```

Comment:
I'll prune the bindings to a minimum. The use case for the main ones is to open related header, source and test files for the current file. I could live without the command that spawns a layout. The historic commands trample on Ubuntu window server - ctrl-alt-t opens a terminal for instance. I was trying to avoid that, and keep h-header, s-source, t-test mnemonics.

Reply:
I've left a minimal set of mappings for file selection operations.